src/utils/createRequire.js

```diff
@@ -0,0 +1,4 @@
+export default (module, disableProcessing) => {
+  const loader = disableProcessing ? '-!' + module : module;
+  return `require (${JSON.stringify(loader)});`;
```

Comment:
What does the JSON.stringify do in this case? Just quote the string? If so, stringify required?